stages/eib_image

```diff
@@ -597,6 +597,22 @@ EOF
     "${EIB_HELPERSDIR}"/create-vm-image "$(eib_outfile ovf.zip)" "${img}"
   fi
 
+  # Create qcow2 image if required.
+  if [ "${EIB_IMAGE_QCOW2}" = true ]; then
+    local qcow2
+
+    qcow2=$(eib_outfile qcow2)
+    qemu-img convert -c -f raw -O qcow2 "${img}" "${qcow2}"
```

Comment:
I recently learned that qcow2 natively supports zstd compression. No idea how good it is but:

```suggestion
    qemu-img convert -c -f raw -O qcow2 -o compression_type=zstd "${img}" "${qcow2}"
```

Reply:
How recently has QEMU/qcow2 supported this? I'd be worried about trying to use the image on an older host that doesn't have zstd support.

Reply:
Since 5.1 https://wiki.qemu.org/ChangeLog/5.1 released in August 2020 https://www.qemu.org/2020/08/11/qemu-5-1-0/.

OK, now I understand that the `-c` above will compress with `zlib`. I incorrectly believed that `-o compression_type=...` is what enabled compression. So I think `zlib` is fine.

Reply:
Indeed. I'll add a comment about exactly what the convert command is doing.